lib/build.api.js

```diff
@@ -70,7 +69,7 @@ class Build {
       let project = this.aquifer.project;
 
       // Set some reasonable defaults.
-      this.options.makeFile = this.getAbsolutePath(this.options.makeFile) || project.absolutePaths.makeFile;
+      this.options.makeFile = project.absolutePaths.make;
```

Comment:
Yeah, I think it's probably a good idea to accept a command option, @JeffTomlinson that looks like the correct ternary. Additionally, we should set the default value of makeFile to false, as noted above, so that we don't get an undefined property error.